client/my-sites/theme/themes-related-card/index.jsx

```diff
@@ -58,8 +59,8 @@ const ThemesRelatedCard = React.createClass( {
 					{ themes.map( theme => (
 						<li key={ theme.id }>
 							<Card className="themes-related-card__card">
-								<a href={ getDetailsUrl( theme ) }>
-									<img src={ theme.screenshot + '?w=' + '660' }/>
+								<a href={ this.props.getDetailsUrl( theme ) }>
+									<img src={ theme.screenshot + '?w=' + '660' } />
```

Comment:
I'm thinking there was maybe a good reason why `'?w=' + '600'` isn't just `'?w=600'`. A comment would probably help here if you have any reason to keep it this way.